opm/input/eclipse/EclipseState/Util/OrderedMap.hpp

```diff
@@ -20,14 +20,67 @@
 #ifndef OPM_ORDERED_MAP_HPP
 #define OPM_ORDERED_MAP_HPP
 
+#include <fmt/format.h>
+
 #include <unordered_map>
 #include <vector>
 #include <string>
 #include <stdexcept>
 #include <iterator>
+#include <string>
+#include <sstream>
+#include <set>
+#include <iostream>
```

Comment:
Please don't include `<iostream>` in a header unless you actually use any of the predefined streams.